### DISCUSSION TITLE
Make `pdk test unit --format=junit:report.xml --verbose` both show rspec documentation and generate junit file

### DISCUSSION THREAD

**Discussion by JohnEricson** [puppetlabs-toy-chest/pct#189] [Ideas]
PDK version: 2.1.1
OS: Windows

I have a Jenkins job that runs `pdk test unit --format=junit:report.xml` on all of my modules. Now I would also like the job to show the rspec documentation format by adding the `--verbose` option to pdk. This to make it easier for people to see the status/documentation of all the tests in the module.
But `--verbose` doesn't show anything when used together with `--format=junit:report.xml`. But it works fine when used with `--format=..` like this `pdk test unit --verbose`.
I would like pdk to be improved so both of these options work together.

**Comment by sanfrancrisko** [comment]
Thanks for raising the issue @JohnEricson 

We are currently working on the new PDK - check out our team's strategy here: https://puppet.com/blog/the-future-of-the-puppet-developer-kit-pdk/

We'll keep this request in mind as we are planning / architecting the new `pdk test` functionality
